libr/core/config.c

```diff
@@ -872,6 +857,22 @@ static int cb_iocache(void *user, void *data) {
 	return true;
 }
 
+static int cb_iopcache (void *user, void *data) {
+	RCore *core = (RCore *) user;
+	RConfigNode *node = (RConfigNode *) data;
+	if ((bool)node->i_value == true) {
+		if (core && core->io)
```

Comment:
put this conditional one level above, so there's no need to check it twice (if/else)